Adjust language, grammar, and small mistakes (#19)
NETWAYS/foreman-training: README.md

Foreman's extensibility is shown by introducing some plugins from basic ones to Katello.

Targeted audience are experienced Linux administrators in need of a provisioning and

> management solution for their systems. Knowledge of Puppet is of advantage but not

Reviewer: ```suggestion
management solution for their systems. Knowledge of Puppet is an advantage but not
```

can be included in your training material by changing 'showoff.json', too. In the same way
remove content you do not want to show or change order if you feel it fits your need better.

> For some notes on setting up the training environment have a look at 'Setup.md'.

Reviewer: ```suggestion
For notes on setting up the training environment see 'Setup.md'.
```
